webrtc.html

```diff
@@ -1316,6 +1393,15 @@
                     method was invoked.</p>
                   </li>
```

Comment:
Since we're doing work in the sync section here, we might want to squeeze in an additional [[isClosed]] check.